core/src/test/java/com/linecorp/armeria/common/util/RateLimitingSamplerTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
```

Comment:
If you forked from Brave, you must retain the copyright header of Brave here.

Reply:
Thank you for review. Fixed in 43557a8.

Reply:
Thank you for review. Fixed in 43557a8.